resources/Materials/TestSuite/pbrlib/surfaceshader/transparency_nodedef_test.mtlx

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0"?>
+<materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
This is a very minor recommendation, but it may be worthwhile to run these new MaterialX documents through "mxupdate.py" before integration, so that their formatting is consistent with other documents in the repository.

Reply:
Good idea. I've run it through.